estate/views/estate_property_offers.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_property_offer_list" model="ir.ui.view">
+            <field name="name">estate.property.offer.view.list</field>
+            <field name="model">estate.property.offer</field>
+            <field name="arch" type="xml">
+                <list editable="bottom" decoration-danger="status == 'refused'" decoration-success="status == 'accepted'">
+                    <field name='price'/>
+                    <field name='partner_id'/>
+                    <!-- <field name='status'/> -->
```

Comment:
```suggestion
```
In general, we try to use comments to add context to the code. This doesn't seem to add much.